docs/docs/index.html

```diff
@@ -845,7 +845,7 @@ <h3 class='fl m0' id='clearfix'>
     </h3>
     
     
-      <a class='fr fill-darken0 round round pad1x quiet h5' href='https://github.com/styled-components/polished/blob/c5ccb3bf1c210d26eb3c2a3b7ab71a817b49e7b3/src/mixins/clearFix.js#L26-L35'>
+      <a class='fr fill-darken0 round round pad1x quiet h5' href='https://github.com/iamakulov/polished/blob/9a57c85ee212dd49b3736dcd65b49d75c0a3ea0c/src/mixins/clearFix.js#L26-L35'>
```

Comment:
Not sure about these changes. It that because of a different `origin`? Should I fix it?